passes/cmds/chformal.cc

```diff
@@ -216,6 +274,16 @@ struct ChformalPass : public Pass {
 				}
 
 				for (auto cell : constr_cells)
+				{
+					if (is_triggered_check_cell(cell)) {
+						cell->setPort(ID::TRG, SigSpec());
```

Comment:
This looks wrong. We are simply losing the triggers for `-early`

Reply:
Discarding the triggers is intended and corresponds to the behavior `chformal -early` has prior to these changes (where it would discard a FF no matter what signal the FF is using as a clock) or with these changes after running `async2sync`. It's only valid for a design with a single clock and no async signals, in which case the trigger is the clock and removing the trigger makes it active one clock cycle earlier. For any other kind of design you'd need to run `async2sync` or `clk2fflogic` before running `chformal -early`.

I'm not at all sure now, though, whether my current handling of the `EN` signal is correct and would want to add some tests that better characterize the behavior of it before and after this PR.

Reply:
We are definitely missing some disclaimers in the help message then.

Reply:
I now have a test (that I'll add to this PR) for checking that `async2sync` and `chformal -early` commute and the code in the PR successfully passes that test.

Reply:
> It's only valid for a design with a single clock and no async signals, in which case the trigger is the clock and removing the trigger makes it active one clock cycle earlier. For any other kind of design you'd need to run async2sync or clk2fflogic before running chformal -early.

If that's the case, let's at least bail here if `TRG_WIDTH > 1`